### PR TITLE
Migrate to native dependabot

### PR DESCRIPTION
**Pull request by sibiraj-s**
Lets Migrate to native dependabot?

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  # Enable version updates for npm
+  - package-ecosystem: "npm"
```
Comment (nickmccurdy) [inline .github/dependabot.yml:4]: I assume this refers to the registry, not the package manager, right? If not we may need to change it to Yarn

**Comment by nickmccurdy** [comment]
What's the difference with what we already have? Would we have to authorize the third party Dependabot app in the organization?

File: .github/dependabot.yml (hunk 1 above)
Comment (sibiraj-s) [reply .github/dependabot.yml:4]: > I assume this refers to the registry, not the package manager.

correct.  it refers to ecosystem, not the manager. package manager is automatically detected.

**Comment by sibiraj-s** [comment]
this is what we have now https://github.com/marketplace/dependabot-preview.
 
`Dependabot is moving natively into GitHub!` blog link https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/.

> Would we have to authorize the third party Dependabot app in the organization?

since dependabot is now native to github, just merging should do. Prettier main repo is already using this

May be the old app `dependabot-preview` has to be disabled (i am not completely sure about this.)

**Pull request merged**